spec/open-api.yml

```diff
@@ -505,6 +509,23 @@ components:
         - config
         - status
         - createdAt
+    PipelineEvent:
+      type: object
+      description: Pipeline Status event.
+      properties:
+        system:
```

Comment:
add an example field, with an example string. Also system can be enumerated string (pod, deployment).